README.md

```diff
@@ -62,7 +62,7 @@ For more information see the [official lakeFS documentation](https://docs.lakefs
    curl https://compose.lakefs.io | docker-compose -f - up
    ```
 
-3. Open [http://127.0.0.1:8000/setup](http://127.0.0.1:8000/setup) in your web browser to set up an initial admin user, used to login and send API requests.
+3. Open [http://127.0.0.1:8000/setup](http://127.0.0.1:8000/setup) in your web browser to set up an initial admin user, log in and send API requests.
```

Comment:
Please also revert this change: it changes meaning.  The admin user is used to login, while after the change it says that surfing to `/setup` is the way to log in.

Reply:
Gentle reminder about this comment.

Reply:
if that's on purpose than it's unclear to me what did you meant here

Reply:
Sorry for lack of clarity.

The `/setup` page is allows you to set up the initial admin user.  This ("initial admin user") is "used to login and send API requests).  But the `/setup` page itself does *not* allow you to log into the system or send API requests.

Here's another attempt at clarity.  There are clearly two steps here (first set up, then log in and send API requests).  And this very conversation is (empirical) proof that the phrasing is confusing!  So I want to split this sentence.  (I don't care about "login" or "log in", so I'll go with Grammarly).
```suggestion
3. Open [http://127.0.0.1:8000/setup](http://127.0.0.1:8000/setup) in your web browser to set up an initial admin user.  You will use this user to log in and send API requests.
```

WDYT?